docs/LibcSupport.md

```diff
@@ -1,27 +1,27 @@
-stdlib.h | Partial | Unsupported functions: div(), imaxabs(), imaxdiv(), ldiv(), lldiv() |
-string.h | Partial | Unsupported functions: strerror(), strsignal() |
-tgmath.h | Partial | Unsupported functions: acosh(), asinh(), fmal(), lgamma(), lgamma_r(), scalbn(), scalbnf(), scalbnl(), sinh(), sinhl(), tgamma() |
-threads.h | Partial | Synchronization primitives are not secure across calls to host. Threads are still scheduled by the untrusted host process and an enclave cannot rely on threads making forward progress. Unsupported functions: pthread_create(), pthread_join() and pthread_detach() |
-time.h | Partial | All time functions implicitly call out to untrusted host for time values. The resulting time values should not be used for security purposes. Supported functions: time(), gettimeofday(), clock_gettime(), nanosleep(). Please note that clock_gettime() only supports CLOCK_REALTIME  |
+stdio.h | Partial | All I/O functions implicitly call out to untrusted host. <br> **Supported functions:** snprintf(), sscanf(),  _vfscanf()*_, vsnprintf(), vsscanf(), sprintf(), vsprintf(), puts(), putchar(), vprintf(), printf(), _fprintf()*_, _getc()*_, _ungetc()*_, _fwrite()*_, _fflush()*_, _fputs()*_, _fputc()*_ <br> _* Only has support for the streams stderr and stdout, and does not set ferror_ |
+stdlib.h | Partial | **Unsupported functions:** div(), imaxabs(), imaxdiv(), ldiv(), lldiv() |
+string.h | Partial | **Unsupported functions:** strerror(), strsignal() |
+tgmath.h | Partial | **Unsupported functions:** acosh(), asinh(), fmal(), lgamma(), lgamma_r(), scalbn(), scalbnf(), scalbnl(), sinh(), sinhl(), tgamma() |
+pthread.h | Partial | Synchronization primitives are not secure across calls to host. Threads are still scheduled by the untrusted host process and an enclave cannot rely on threads making forward progress. <br> **Supported functions:** <br> _- General:_ oe_pthread_self(), oe_pthread_equal(), oe_pthread_once() <br> _- Spinlock:_ oe_pthread_spin_init(), oe_pthread_spin_lock(), oe_pthread_spin_unlock(), oe_pthread_spin_destroy() <br> _- Mutex:_ oe_pthread_mutexattr_init(), oe_pthread_mutexattr_settype(), oe_pthread_mutexattr_destroy(), oe_pthread_mutex_init(), oe_pthread_mutex_lock(), oe_pthread_mutex_trylock(), oe_pthread_mutex_unlock(), oe_pthread_mutex_destroy() <br> _- RW Lock:_ oe_pthread_rwlock_init(), oe_pthread_rwlock_rdlock(), oe_pthread_rwlock_wrlock(), oe_pthread_rwlock_unlock(), oe_pthread_rwlock_destroy() <br> _- Cond:_ oe_pthread_cond_init(), oe_pthread_cond_wait(), oe_pthread_cond_timedwait(), oe_pthread_cond_signal(), oe_pthread_cond_broadcast(), oe_pthread_cond_destroy() <br> _- Thread local storage:_ oe_pthread_key_create(), oe_pthread_key_delete(), oe_pthread_setspecific(), oe_pthread_getspecific() |
```

Comment:
RE: pthread.h, we need to remove the "oe" prefixes since the oe versions are not public.

Reply:
Thanks for the catch, I'm too used to using corelibc now!